test/boilerplate/test-karma.js

```diff
@@ -1,6 +1,6 @@
 var shell = require('shelljs');
 var util = require('util');
-var defaultArgs = '--single-run --browsers=Safari,ChromeHeadless,FirefoxHeadless';
+var defaultArgs = '--single-run --browsers=ChromeHeadless,FirefoxHeadless';
```

Comment:
Why is this being removed?

Reply:
safari has issues with karma, it was removed from core a while back but since we haven't updated optimizely in some time, it was never removed here